src/app/features/profile/store/profile.reducers.ts

```diff
@@ -23,6 +24,11 @@ const reducer = createReducer(
 
         return { ...state, user: dummyProfile };
 
+    }),
+    on(loadProfileSuccess, (state, action) => {
+
+        return { ...state, user: action.user };
```

Comment:
not sure how are you maintaining users list in state if each loadProfileSuccess action will overwrite user property from a state
you should have add users property in the state to store the whole list